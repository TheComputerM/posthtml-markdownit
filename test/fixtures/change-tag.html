<md tag="section">
  # A markdown heading
</md>
