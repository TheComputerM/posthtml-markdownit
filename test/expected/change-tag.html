<section>
<h1>A markdown heading</h1>
</section>
